include/constants/battle_script_commands.h

```diff
@@ -230,7 +230,6 @@
 #define VARIOUS_SET_SKY_DROP                         138
 #define VARIOUS_CLEAR_SKY_DROP                       139
 #define VARIOUS_SKY_DROP_YAWN                        140
-#define VARIOUS_JUMP_IF_CANT_FLING                   141
```

Comment:
Instead of deleting can you relabel to VARIOUS_UNUSED_141 or similar?

Reply:
I think we wanted to remove them entirely (and we should've stopped people from PRing them once we got the callnative stuff but we didn't xD), so no point on renaming them to a variable that we won't use imo.

Reply:
Since we want to replace them completely for callnatives, I don't see the usefulness of replacing it with an unused value that we're not gonna use ever.

Reply:
(why did this get commented twice?) I thought the first time didn't go, so I had to rewrite my comment